### PR TITLE
Searchpage

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Created search indices page
image

### CODE REVIEW

File: pages/searchpage.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+
+.form-input{
+    border-radius: 10px;
+    padding: 15px;
+    font-size: 3rem;
+    border: solid var(--grey);
+    
+}
+.text{
```
Comment (Varun-28) [inline pages/searchpage.css:10]: looks clean and ready to merge 👍

File: pages/searchpage.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+
+.form-input{
```
Comment (Hamdrive) [inline pages/searchpage.css:3]: Format code using Prettifier to remove unneccesarry spaces

**Review by Hamdrive — COMMENTED** [review]
Code looks superb, and site looks great as well
Suggest to format code using a formatter

File: pages/searchpage.css (hunk 2 above)
Comment (AnuragSahu11) [reply pages/searchpage.css:3]: Yes will keep in mind from now on

**Pull request merged**